### PR TITLE
Update setup_webapp.html

### PR DESCRIPTION
**Pull request by V4M0N0S**
a clearer description of the process / error

### CODE REVIEW

File: web/templates/pages/setup_webapp.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
 				<?php if( !$WebappInstaller->isDomainRootClean()): ?>
 					<span class="alert alert-info alert-with-icon">
 						<i class="fas fa-info"></i>
-						<?=_('Data loss warning!<br>Your web domain already has files uploaded, the installer will overwrite your files and/or the installation might fail.<br/><br/> Please use the installer only for empty web domains');?>
+						<?=_('Data loss warning!<br>HESTIA found some files in your domain folder. The installer will overwrite your files and / or the installation might fail.<br/><br/> Please use the installer only for empty domain folders!');?>
```
Comment (jaapmarcus) [inline web/templates/pages/setup_webapp.html]: Ideally you want to show:

Data loss warning!
/home/user/web/domain/public_html/ is not empty. The installer will overwrite your files and / or the installation might fail.<br/><br/> Please use the installer only for empty domain folders!');?>

Comment (V4M0N0S) [reply web/templates/pages/setup_webapp.html]: I thought that clients do not have access to home but only to their directory, it makes the most sense.

What do you think?

Comment (jaapmarcus) [reply web/templates/pages/setup_webapp.html]: It depends if they login with Additional ftp or sftp or file manager 

In both last cases the are able to /web/domain/public_html/ 

In the ftp case it depends on how it is set up by default the user get access to /public_html/

**Comment by V4M0N0S** [comment]
Check stopped from @jaapmarcus

**Comment by V4M0N0S** [comment]
Its updates now: https://github.com/hestiacp/hestiacp/pull/2700/commits/98fab667b5b86dbd404c507dbf3ba3cadbb45d1e

**Review by jaapmarcus — COMMENTED** [review]
domain.com needs to be replaced with %s and replaced with the correct domain

**Comment by jaapmarcus** [comment]
Screenshot 2022-07-02 at 12 32 35

**Comment by jaapmarcus** [comment]
Replaced by linked PR

**Pull request closed without merging**